ui-v2/tests/acceptance/dc/services/show/upstreams.feature

```diff
@@ -0,0 +1,47 @@
+@setupApplicationTest
+Feature: dc / services / show / upstreams
+  Background:
+    Given 1 datacenter model with the value "dc1"
+    And 1 node models
+    And 1 service model from yaml
+    ---
+    - Service:
+        Name: ingress-gateway-1
+        Kind: ingress-gateway
+    ---
+  Scenario: Seeing the Linked Services tab
+    When I visit the service page for yaml

```

Comment:
Copy/pasta! Not a biggie but whilst you are in here, you may aswell change this to say 'Upstreams' rather than 'Linked Services' (below here also)